zqd/search/search.go

```diff
@@ -65,20 +66,16 @@ func Search(ctx context.Context, s *space.Space, req api.SearchRequest, out Outp
 	if err != nil {
 		return err
 	}
-	return run(mux, out)
-}
-
-func Copy(ctx context.Context, w []zbuf.Writer, r zbuf.Reader, prog string) error {
-	p, err := zql.ParseProc(prog)
-	if err != nil {
-		return err
+	d := &driver{
+		output:    out,
+		startTime: nano.Now(),
 	}
-	mux, err := zdriver.Compile(ctx, p, r, false, nano.MaxSpan, zap.NewNop())
-	if err != nil {
+	d.start(0)
+	if err := zdriver.Run(mux, d, DefaultStatsInterval); err != nil {
```

Comment:
```suggestion
	if err := driver.Run(mux, d, DefaultStatsInterval); err != nil {
```
(And undo the package rename to zdriver in the imports.)

Reply:
👍 (renamed the driver struct to searchdriver)